get_instagram.py

```diff
@@ -0,0 +1,27 @@
+import requests
+
+BASE_URL = 'https://api.instagram.com/v1/'
```

Comment:
Why didn't you use Python wrapper for Instagram API?

Reply:
done